src/extensions/step_handlers/wim_handler/src/format_drive.cpp

```diff
@@ -17,8 +18,36 @@
         }                    \
     }
 
-HRESULT FormatDrive(char driveLetter)
+static HRESULT DoesVolumeLabelExist(char driveLetter, const std::string& volumeLabel)
```

Comment:
i think this would be more useful as:

`std::string GetVolumelabel(char driveLetter);`

Reply:
and have the caller do the compare.